front-end/src/renderer/pages/Settings/components/KeysTab.vue

```diff
@@ -47,15 +48,20 @@ enum Tabs {
 /* State */
 const isDeleteModalShown = ref(false);
 const isUpdateNicknameModalShown = ref(false);
+const deleteAll = ref(false);
+const isSelectAll = ref(false);
+const selectedKeyPairIdsToDelete = ref<string[]>([]);
+const selectedMissingKeyPairIdsToDelete = ref<string[]>([]);
+const selectMany = ref(false);
 
 const decryptedKeys = ref<{ decrypted: string | null; publicKey: string }[]>([]);
 const publicKeysPrivateKeyToDecrypt = ref('');
-const keyPairIdToDelete = ref<string | null>(null);
 const keyPairIdToEdit = ref<string | null>(null);
-const missingKeyPairIdToDelete = ref<number | null>(null);
 const currentTab = ref(Tabs.ALL);
 const isDeletingKey = ref(false);
 const selectedRecoveryPhrase = ref<string>('');
+const deleteSingleLocal = ref<string[]>([]);
```

Comment:
These two state variables `deleteSingleLocal` and `deleteSingleMissing` are unnecessary.
You can use the `selectedKeyPairIdsToDelete` and `selectedMissingKeyPairIdsToDelete`.
The `single` state variables are assigned in `handleDeleteModal` and `handleMissingKeyDeleteModal`, there you may use the `selected` state variables